### DISCUSSION TITLE
`Image`s working with v 10.0.0  don't work as expected with v 10.0.2

### DISCUSSION THREAD

**Discussion by jahirfiquitiva** [vercel/next.js#19228] [Help]
# Bug report

## Describe the bug

I updated my website to v 10.0.0 right after it was published, now, while trying to update to v 10.0.1 most of images got broken, sized incorrectly and some styles don't work the same.

## To Reproduce

Please go to the [production website](https://jahir.dev) and [testing website](https://web-git-next-1001.jahirfiquitiva.vercel.app/) for comparison. Here's screenshots for troubling images comparison.

| Before | After |
|:------:|:-----:|
|<img width="206" alt="Shot 2020-11-16 at 22 05 34@2x" src="https://user-images.githubusercontent.com/10360816/99341438-dd992880-2857-11eb-981b-fcfbf98dc0c1.png">|<img width="199" alt="Shot 2020-11-16 at 22 05 57@2x" src="https://user-images.githubusercontent.com/10360816/99341456-ea1d8100-2857-11eb-83ff-df52b185b384.png">|

[production website](https://jahir.dev) vs [testing website](https://web-git-next-1001.jahirfiquitiva.vercel.app)
| Before | After |
|:------:|:-----:|
|<img width="199" alt="Shot 2020-11-19 at 13-05-08" src="https://user-images.githubusercontent.com/10360816/99705750-28898a80-2a68-11eb-9247-4b8cb0e13044.png">|<img width="206" alt="Shot 2020-11-19 at 13-04-46" src="https://user-images.githubusercontent.com/10360816/99705741-258e9a00-2a68-11eb-8e85-bd886237f48e.png">|

[production website](https://jahir.dev/thanks) vs [testing website](https://web-git-next-1001.jahirfiquitiva.vercel.app/thanks)
| Before | After |
|:------:|:-----:|
|<img width="336" alt="Shot 2020-11-16 at 22 06 51@2x" src="https://user-images.githubusercontent.com/10360816/99341542-16d19880-2858-11eb-9600-e3ce78f944d0.png">|<img width="304" alt="Shot 2020-11-16 at 22 06 58@2x" src="https://user-images.githubusercontent.com/10360816/99341546-189b5c00-2858-11eb-979c-ce05e3ffffd3.png">|

[production website](https://jahir.dev/error) vs [testing website](https://web-git-next-1001.jahirfiquitiva.vercel.app/error)
| Before | After |
|:------:|:-----:|
|<img width="683" alt="Shot 2020-11-16 at 22 08 21@2x" src="https://user-images.githubusercontent.com/10360816/99341642-47b1cd80-2858-11eb-8731-6b0fc2f64496.png">|<img width="684" alt="Shot 2020-11-16 at 22 08 28@2x" src="https://user-images.githubusercontent.com/10360816/99341650-4aacbe00-2858-11eb-95ad-fdd0b80cfb93.png">|

## Expected behavior

All images should look and work the same after update.

## System information

- OS: macOS
- Version of Next.js: 10.0.2
- Version of Node.js: 12.16.2

**Comment by timneutkens** [comment]
It seems like you've added custom styling to your application that changes the way `next/image` was sized which is causing issues after upgrading as the layout system got changed a bit to accommodate all use cases people were running into. You'll want to pick the right layout option for your particular layout here: https://nextjs.org/docs/api-reference/next/image#layout
